apps/swirl-docs/src/components/HeaderNavigation/index.tsx

```diff
@@ -76,21 +76,51 @@ const HeaderNavigation = () => {
                   ? "before:opacity-100"
                   : "before:opacity-0"
               }`}
-            >
-              <Link
-                className={`text-text-default text-base ${
-                  activePath?.includes(link.url) ? "text-border-info" : ""
-                }`}
-                href={link.url}
+                >
+                  <Link
+                    className={`text-text-default text-base ${
+                      activePath?.includes(link.url) ? "text-border-info" : ""
+                    }`}
+                    href={link.url}
+                  >
+                    <a>{link.title}</a>
+                  </Link>
+                </li>
+              ))}
+            </ul>
+          </nav>
+        </header>
+      </DesktopView>
+      <MobileView>
+        <header className="sticky top-0 flex flex-col w-full h-full max-h-screen bg-background-default">
+          <div
+            aria-label="main"
+            className="flex justify-between items-center h-16 w-full px-4 border-b-1 font-normal text-base"
+          >
+            <HeaderLogo />
+            {isMobileNavOpen ? (
+              <button type="button" onClick={() => setIsMobileNavOpen(false)}>
```

Comment:
**suggestion:** For better accessibility, the toggle should be inside the navigations `nav` tags (or set `aria-controls` on the button, if not possible). You could also set `aria-expanded="true|false"`. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded#menu